docs/framework/data/adonet/performance-counters.md

```diff
@@ -391,7 +391,7 @@ class Program
 ## See also
```

Comment:
can you fix a few issues I'm seeing above as well?
there are several lines that say retrive instead of retrieve 

also on line 378 can you remove the two // to uncomment that line of code? I think that's what causing the weird formatting here on GitHub on the see also section